lib/account_deleter.rb

```diff
@@ -17,8 +17,8 @@ class AccountDeleter
 
   attr_accessor :person, :user
 
-  def initialize(diaspora_handle)
-    self.person = Person.where(:diaspora_handle => diaspora_handle).first
+  def initialize(person)
```

Comment:
Why? I'd rather not pass objects around workers... Why don't you use the guid here?